Cris/ledFade/ledFade.ino

```diff
@@ -0,0 +1,24 @@
+
+//declare the variable: my pin number, initial brighness is set to 0, and the fade anount can be adjusted
+int led = 11;   
+int fadeAmount = 5;
+int brightness = 0;        
+
+//declare the function that will set the pin to output mode
+void setup() {
+  pinMode(led, OUTPUT);
+  Serial.begin(9600);  
```

Comment:
Would be good to state what Serial does (for others reading this that may have completely forgotten)